fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -947,7 +947,7 @@ public class Config extends ConfigBase {
      * You may reduce this number to avoid Avalanche disaster.
      */
     @ConfField(mutable = true)
-    public static int max_point_query_retry_time = 2;
+    public static int max_point_query_retry_count = 2;
```

Comment:
why change config name?

Reply:
maybe time is more likely for timeout?

Reply:
cahnge config name is a behavior change. If user has already used this configuration, it may cause unnecessary issues

Reply:
fixed